pkg/provider/mock.go

```diff
@@ -0,0 +1,16 @@
+package provider
+
+import (
+	"context"
+)
+
+type MockStore struct {
```

Comment:
Could we get a title on the MockStore struct?

Reply:
Changes are done.